public/samples/DataStreams/StreamsUpkeep.sol

```diff
@@ -110,65 +98,48 @@ contract StreamsUpkeep is ILogAutomation, StreamsLookupCompatibleInterface {
         return (true, abi.encode(values, extraData));
     }
 
-    function performUpkeep(bytes calldata performData) external override {
+    // function will be performed on-chain
+    function performUpkeep(bytes calldata performData) external {
+        // Decode incoming performData
         (bytes[] memory signedReports, bytes memory extraData) = abi.decode(
             performData,
             (bytes[], bytes)
         );
 
         bytes memory report = signedReports[0];
```

Comment:
to help educate, this `report` could be called `unverifiedReport` to verify its status.  Will help people understand the stages/steps of a report.

Reply:
Done